src/mount/writedata.cc

```diff
@@ -215,6 +215,7 @@ static InodeDataMap inodedataMap;
 
 static uint32_t gWriteWindowSize;
 static uint32_t gChunkserverTimeout_ms;
+static std::atomic<bool> gIgnoreFlush;
```

Comment:
Outside of registering it for tweaks, it doesn't seem this variable is used elsewhere? I'm assuming it's updated when the tweaks file is updated, but will this be used to determine whether to flush or not?

Reply:
Yes, you are right. The variable was not correctly being updated when its value changed in the tweaks file, the solution for this was to set the `gIgnoreFlush` variable in the tweaks in the `init` function.